java/src/main/native/include/jni_utils.hpp

```diff
@@ -28,6 +28,7 @@
 #include "cudf/legacy/table.hpp"
 #include "utilities/column_utils.hpp"
 #include "cudf/utilities/error.hpp"
+#include "utilities/legacy/error_utils.hpp"
```

Comment:
Include only change in this file?

Reply:
yes CUDA_TRY moved files and was breaking the build.